db/migrate/1730785487527_create_oauth_access_tokens.rb

```diff
@@ -0,0 +1,20 @@
+class CreateOauthAccessTokens < ActiveRecord::Migration[6.0]
+  def change
+    create_table :oauth_access_tokens do |t|
+      t.references :resource_owner_id, null: false
+      t.string :token, null: false, index: { unique: true }
```

Comment:
The 'resource_owner_id' and 'application_id' columns should use 'foreign_key: true' in the migration to ensure referential integrity with the associated tables.